### DISCUSSION TITLE
Just looking at the changes in app.js, then stumbled upon this :)

### DISCUSSION THREAD

**Discussion by the-skiddo** [QuiteAFancyEmerald/InvisiProxy#64] [Q&A] [answered]
I got a big laugh from this, but I have no idea what the list of random food-related sentences/items were about. Where can you find this while in the actual website? (btw some of these sentences actually sound really good/helpful) 

Code I was looking at: 

cookingInserts = [
		"Cooking is something that most of people are doing it or try to do it. Some do it as a profession, some do it for fun, and some do it because they have to do it. People are going to cooking, because some food is raw that people cannot eat it, because it is not tasty.",
		"Olive Garden opened in 1982 specializing in Italian based food. They have pasta, breadsticks, and soup. Zuppa Toscana is a soup that is one of their most popular soups. It’s spicy kick, and creamy tasting is the best part about it. This recipe is very easy to make, but is the best at Olive Garden.",
		"Homemade chicken avocadoes lime soup, doesn’t that sound Delicious! This simple, yet filling dish can serve a family of six. The preparation time takes roughly fifteen minutes and cook time is approximately twenty minutes.",
		"Let’s discuss their Food Quality. Their very well known for their wide mixture of pizzas from pepperoni, pepperoni and sausage, Colby classic, and bourbon street.",
		"The dinner menu I will be preparing to celebrate a special occasion will include: a nice juicy steak, fresh corn on the cob, hash brown with bacon casserole, and for the dessert peanut butter pie.",
		"Enter the spice aisle and grab one bottle of onion powder and one of... hamburger seasoning. Also, go to the vegetable area and pick up one regular onion. Then, walk back to the meat section and pick up a package of ground meat. Go to the bread aisle and grab a pack of the original Hawaiian sub-rolls and a pan that the rolls will fit into. Everybody knows that cheese is what tops off a good cheeseburger slider, so go pick up a pack of the favorite type of.",
		"Entertaining for Thanksgiving can be very rewarding, and very exhausting. Over the years I have found that planning and preparing for the big day is essential. I love to cook. I enjoy preparing a wonderful Thanksgiving meal for my loved ones.",
		"Peel tomatoes with ease! Cut an X in the top, and then simmer in a pot of hot water for 15 to 30 seconds. Cool down and the skin will fall right off.",
		"To create an egg wash, whisk together a large egg with one tablespoon of water until smooth. Use as a glue to seal pastries, then brush on top for a glossy appearance.",
		"Embrace salt. Don’t be afraid to use salt; it pulls the flavors out of your dishes. Cook with kosher salt and season with sea salt.",
		"For a great hardboiled egg every time, bring your pot to a boil and then turn off the heat. Let your eggs sit in the heated pot for 12 minutes and then transfer to cold water."
		"This is a cool example cooking sentence.",
		"I wish I could boil hot water. Try to fill this sentence with a lot of cooking related keywords."
	],
	vegetables = ['Beet', 'Radish', 'Potato', 'Yam', 'Carrot', 'Garlic', 'Onion', 'Asparagus', 'Rhubarb', 'Celery', 'Lettuce', 'Spinach', 'Kale', 'Cabbage', 'Arugula', 'Broccoli', 'Artichoke', 'Cauliflower', 'Tomato', 'Avocado', 'Pepper', 'Squash', 'Pumpkin', 'Zucchini']

``

**Comment by QuiteAFancyEmerald** [comment, marked as answer]
None of these changes are physically visible on the site lol but the use of `<span>` filled with random cooking keywords is used to prevent filters from source blocking the entire site but also helps with categorization somewhat, especially since it's randomized. 

(This is a change you can see when you are viewing the source and reload the page.)
